src/fsharp/NicePrint.fs

```diff
@@ -103,7 +103,7 @@ module internal PrintUtilities =
         let tcref = attrib.TyconRef
         squareAngleL (layoutTyconRefImpl true denv tcref)
 
-module private PrintIL = 
```

Comment:
Unfortunately private modules are not yet properly supported in Fable. I'll open an issue to discuss it.